SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.h

```diff
@@ -22,3 +23,5 @@ void GenerateSubobjectSchema(FComponentIdGenerator& IdGenerator, UClass* Class,
 
 // Generates schema for RPC endpoints.
 void GenerateRPCEndpointsSchema(FString SchemaPath);
+
+void AddComponentId(const Worker_ComponentId ComponentId, Worker_ComponentId (&SchemaComponents)[ESchemaComponentType::SCHEMA_Count], const ESchemaComponentType ComponentType);
```

Comment:
Can we typedef the c array?

Reply:
Can we using the c array?

Reply:
What is it we want here? A typedef for arrays of component IDs of this exact length? Where would the best place be for this? `SpatialConstants`?

Reply:
I believe we can using a `using`. These normally go in `SpatialCommonTypes`, however if usage is limited to this file then can just keep it locally.

Reply:
Introduced a using alias for the array in this file